File: BOJ/BOJ_14467.js
const fs = require('fs')
const stdin = fs.readFileSync('/dev/stdin').toString().split('\n')
const input = (() => {
  let line = 0
  return () => stdin[line++]
})()

const N = Number(input())
const cows = Array(11).fill(-1)

let cnt = 0
for (let i = 0; i < N; i++){
    let [a, b] = input().split(' ').map(Number)

    if (cows[a] == -1) {
        cows[a] = b
    }
    else {
        if (cows[a] != b) {
            cnt += 1
            cows[a] = b
        }
    }
}

console.log(cnt)

File: BOJ/BOJ_14467.py
from sys import stdin

T = int(stdin.readline().rstrip())
cows = [-1] * 11
count = 0

for i in range(T):
    N, M = map(int, stdin.readline().split())
    if cows[N] == -1:
        cows[N] = M
    else:
        if cows[N] != M:
            count += 1
            cows[N] = M

print(count)